internal/telemetry/segment.go

```diff
@@ -96,7 +95,8 @@ func LogShellDurationEvent(eventName string, startTime string) error {
 	// Ignore errors, telemetry is best effort
 	_ = segmentClient.Enqueue(segment.Track{
 		AnonymousId: evt.AnonymousID,
-		Event:       evt.eventName,
+		// Event name. We trim the prefix from shell-interactive/shell-ready to avoid redundancy.
+		Event: fmt.Sprintf("[%s] Shell Event: %s", evt.AppName, strings.TrimPrefix(eventName, "shell-")),
```

Comment:
Are you sure this is what we want?

Wouldn't this add "Shell Event" to *all* events including the "Command" events that already exist?

I would have thought instead that you need to change the string wherever the `shell-interactive` event is triggered.

Reply:
no, those command events are still logged in `midcobra/telemetry.go`.
I have a cleanup I want to do to unify these further, but have held back since its a bit involved.